craft_cli/helptexts.py

```diff
@@ -353,7 +360,8 @@ def _build_markdown_command_help(self, usage, overview, options, other_command_n
             "|-|-|",
         ]
         for title, text in options:
-            option_lines.append(f"| `{title}` | {text} |")
+            _text = text.replace("\n", "<br>")
+            option_lines.append(f"| `{title}` | {_text} |")
```

Comment:
what happens to the `R|` prefix when showing as markdown?

Reply:
Oh, that should be removed.

Reply:
Ok, we should test that the `"R|"` prefix gets removed in the markdown case then (that's probably what the coverage is complaining